build.gradle

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java project to get you started.
+ * For more details take a look at the Java Quickstart chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/5.2/userguide/tutorial_java_projects.html
+ */
+plugins {
+    // Apply the java plugin to add support for Java
+    id 'java'
+
+    // Apply the application plugin to add support for building an application
+    id 'application'
+
+    id 'base'
+}
+
+description = 'Administration console bootstrapper'
+version = '1.0'
+
+repositories {
+    // Use jcenter for resolving your dependencies.
+    // You can declare any Maven/Ivy/file repository here.
+    //jcenter(),
+    mavenCentral()
+}
+
+dependencies {
+    // This dependency is found on compile classpath of this component and consumers.
+    implementation 'com.google.guava:guava:27.0.1-jre'
+
+    // Use TestNG framework, also requires calling test.useTestNG() below
+    testImplementation 'org.testng:testng:6.14.3'
```

Comment:
hmmm, could we perhaps use jUnit or Spock?

Reply:
This was just in the default "gradle init" setup.  However the Equella autotests are using testNG also anyway.

Reply:
Gotcha.

But yes, although oEQ uses TestNG, I feel newer stuff should probably go with what seems to more common jUnit. And then if people are keen, I'd go Spock - but I appreciate this is then different.

Reply:
Have switched to JUnit in my working branch.